app/javascript/app/utils/ms-formatter.js

```diff
@@ -1,18 +1,16 @@
-function formatMinutes(minutes) {
-  return minutes || 0;
-}
+const formatTime = (time) => `0${time}`.slice(-2);
 
-function formatSeconds(seconds) {
-  return seconds < 10 ? `0${seconds}` : seconds;
-}
-
-export default (milliseconds) => {
+export default (milliseconds, screenReader) => {
   const seconds = milliseconds / 1000;
   const minutes = parseInt(seconds / 60, 10);
   const remainingSeconds = parseInt(seconds % 60, 10);
 
-  const displayMinutes = formatMinutes(minutes);
-  const displaySeconds = formatSeconds(remainingSeconds);
+  const displayMinutes = formatTime(minutes);
+  const displaySeconds = formatTime(remainingSeconds);
+
+  const displayTime = screenReader
+    ? `00:${displayMinutes}:${displaySeconds}`
+    : `${minutes > 0 ? `${displayMinutes} minute(s) and ` : ''}${displaySeconds} second(s)`;
```

Comment:
Unfortunately this will be a bit more complicated, because we need to support non-English languages as well. Also, it would be nice if the text would automatically adapt to include/exclude the plural inflection (minute vs. minutes, second vs. seconds).

We might be able to use pieces of the [`dotiw` gem](https://github.com/radar/distance_of_time_in_words) or built-in Rails strings for this.

Maybe roughly something like:

```js
/**
 * @typedef {typeof window & {
 *   LoginGov: { I18n: import('@18f/identity-i18n').I18n } }
 * } GlobalWithLoginGov
 */

const { t } = /** @type {GlobalWithLoginGov} */ (window).LoginGov.I18n;

const minutesKey = displayMinutes === 1 ? 'one' : 'other';
const secondsKey = displaySeconds === 1 ? 'one' : 'other';

const displayTime = screenReader
  ? `00:${displayMinutes}:${displaySeconds}`
  : t(`datetime.dotiw.minutes.${minutesKey}`).replace('%{count}', displayMinutes) + t('datetime.dotiw.two_words_connector') + t(`datetime.dotiw.seconds.${secondsKey}`).replace('%{count}', displaySeconds);
```